RocketChat/Rocket.Chat
# [FIX] Fixing the Channel List UI

## Proposed changes (including videos or screenshots)

The channel list on the side navbar gets cut at the bottom and hence some part of last channel gets hidden behind the logo
Before : 


![initial](https://user-images.githubusercontent.com/56491104/110130822-78117d80-7def-11eb-9c74-33f7410ee2cf.png)


After 

![after](https://user-images.githubusercontent.com/56491104/110130866-85c70300-7def-11eb-82ad-fd6b65220cd9.png)






## Issue(s)

#20994 
## Steps to test or reproduce
Just go to create many channels and you will see at the end that the last channel gets cut off
## Further comments
